libs/application/templates/party-letter/src/fields/EndorsementList/EndorsementTable.tsx

```diff
@@ -1,6 +1,6 @@
 import React, { FC } from 'react'
 import { Application } from '@island.is/application/core'
-import { Endorsement } from '../../types'
+import { Endorsement } from '../../lib/dataSchema'
```

Comment:
This might not be what we want since this takes types from our current system and not the source system